python/openai/openai_frontend/main.py

```diff
@@ -120,6 +120,18 @@ def parse_args():
         default="0.0.0.0",
         help="Address/host of frontends (default: '0.0.0.0')",
     )
+    triton_group.add_argument(
+        "--tool-call-parser",
+        type=str,
+        default=None,
+        help="Specify the parser for handling tool calling related response text. Options include: 'llama3' and 'mistral'.",
+    )
+    triton_group.add_argument(
+        "--chat-template",
+        type=str,
+        default=None,
+        help="The path of the jinja chat template file.",
+    )
```

Comment:
Why do we need an extra chat template arg? Just for supporting models with no built-in chat template? Are we using that in a test or example somewhere?